apps-client/mobile/app/(tabs)/profile/index.tsx

```diff
@@ -173,21 +171,21 @@ const HeaderSection = () => {
 const ProfileSection = () => {
   const router = useRouter()
   const profileQuery = reactQueryClient.useQuery('/profile/me', {})
-  const getRoleName = (role: UserRole) => {
-    switch (role) {
-      case 'REPRESENTATIVE':
-        return 'สมาชิกสส.พรรคประชาชน'
-      case 'USER':
-        return 'ประชาชน'
-      case 'STAFF':
-        return 'สมาชิกพรรค'
-      case 'OFFICIAL':
-        return 'คณะทำงาน'
-      case 'MEMBER':
-        return 'สมาชิกพรรค'
-      default:
-        exhaustiveGuard(role)
+  const getRoleName = (roles: string[]) => {
+    for (const role of roles) {
+      if (role.startsWith('pple-member:member')) return 'สมาชิกพรรค'
+      switch (role) {
+        case 'pple-ad:mp':
+          return 'สมาชิกสส.พรรคประชาชน'
+        case 'pple-ad:hq':
+          return 'ส่วนกลาง'
+        case 'pple-ad:province':
+        case 'pple-ad:local':
+          return 'คณะทำงาน'
+      }
```

Comment:
can we write down all possible roles (or some that we've seen) somewhere
like what about 'official'

Reply:
Pinned in discord 👍 👍